typhon/cloudmask/aster.py

```diff
@@ -617,79 +619,100 @@ def sensor_angles(self, sensor="vnir"):
              Algorithm Theoretical Basis Document for ASTER Level-1 Data 
              Processing (Ver. 3.0).1996.
         """
+        if channel is not "3B":
+            sensor = self.channel2sensor[channel]
+        else:
+            sensor = "VNIRB"
+
         # Angular data from ASTER metadata data.
         S = float(self.meta["MAPORIENTATIONANGLE"])
 
-        if sensor == "vnir":
-            P = float(self.meta["POINTINGANGLE.1"])
-            FOV = 6.09  # Field of view
-            # IFOV = np.rad2deg(21.3e-6) Instantaneous field of view
-            field = self.get_radiance(channel="1")
-        elif sensor == "swir":
-            P = float(self.meta["POINTINGANGLE.2"])
-            # IFOV = np.rad2deg(42.6e-6)
-            FOV = 4.9
-            field = self.get_radiance(channel="4")
-        elif sensor == "tir":
-            P = float(self.meta["POINTINGANGLE.3"])
-            # IFOV = np.rad2deg(127.8e-6)
-            FOV = 4.9
-            field = self.get_radiance(channel="10")
+        FOV = {"VNIR": 6.09,
+               "VNIRB": 5.19,
+               "SWIR": 4.9,
+               "TIR":4.9
+              }
```

Comment:
Picky comment from my side ;) There is two options to format code like this:
1. If you know that the dictionary is complete:
```python 
FOV = {"VNIR": 6.09, "VNIRB": 5.19,  "SWIR": 4.9, "TIR": 4.9}
```
2. If it may be extended in the future:
```python 
FOV = {
    "VNIR": 6.09,
    "VNIRB": 5.19, 
    "SWIR": 4.9,
    "TIR": 4.9,
}
```